SI/Sem.05/Pract.05/Task09.cpp

```diff
@@ -0,0 +1,77 @@
+#include <iostream>
+
+void print(char values[3][3]) {
```

Comment:
Use these two cool lines to clear your console before every print
    std::cout << "\033[;H"; // Moves cursor to the top left
    std::cout << "\033[J"; // Clears the console